### ISSUE TITLE
Selection options dropdown (non-modal) - keyboard navigation (keys up / down) does not work.

### ISSUE THREAD

**Issue by awf1234** [KevinVandy/material-react-table#1428]
### material-react-table version

3.2.1

### react & react-dom versions

19.0.0

### Describe the bug and the steps to reproduce it

Example: your examples, e.g. Editing CRUD Examples https://www.material-react-table.com/docs/examples/editing-crud-inline-row:
In the selection drop-down on the State column keyboard navigation (with keys up / down) does not work. It works in the modal example (CRUD Modal Editing) on the modal screen, but not in the non-modal examples. 
I am looking at this on a Linux Ubuntu desktop computer. 

### Minimal, Reproducible Example - (Optional, but Recommended)

The Editing CRUD examples at your web page - selection drop-down for column 'State':
Non-modal:   https://www.material-react-table.com/docs/examples/editing-crud-inline-row:
... keyboard navigation does not work. 
Modal:  https://www.material-react-table.com/docs/examples/editing-crud
... key board navigation works.
  

### Do you intend to try to help solve this bug with your own PR?

None

### Terms

- [x] I understand that if my bug cannot be reliably reproduced in a debuggable environment, it will probably not be fixed and this issue may even be closed.

**Comment by PMudra** [comment]
Same for the [Row Actions Menu](https://www.material-react-table.com/docs/guides/row-actions#row-actions-menu-items-demo).

Keyboard navigation inside the menu seems to be working when disabling `enableKeyboardShortcuts` option.
